### PR TITLE
{math}[system/system] MCR vR2021b

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by robert-mijakovic** [comment]
Test report by @robert-mijakovic
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
mel0385 - Linux centos linux 8.4.2105, x86_64, AMD EPYC 7H12 64-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/a6e2af3a8c62f598cb530419dba79632 for a full test report.

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/MCR/MCR-R2021b.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'MCR'
+version = 'R2021b'  # runtime version 9.10
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/MCR/MCR-R2021b.eb:5]: shouldn't the comment be changed to 9.11?

Comment (boegel) [reply easybuild/easyconfigs/m/MCR/MCR-R2021b.eb]: fixed in 9b13324

Comment (robert-mijakovic) [reply easybuild/easyconfigs/m/MCR/MCR-R2021b.eb]: @boegel thank you.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3103.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/1fe668f1c9a7fbdbab6335fe2d7e714b for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @robert-mijakovic!

**Pull request merged**